src/core/config.py

```diff
@@ -56,6 +56,7 @@ def get_ca_verify_setting():
 PROMETHEUS_URL = os.getenv("PROMETHEUS_URL", "http://localhost:9090")
 TEMPO_URL = os.getenv("TEMPO_URL", "http://localhost:8080")
 LLAMA_STACK_URL = os.getenv("LLAMA_STACK_URL", "http://localhost:8321/v1/openai/v1")
+LLAMA_STACK_CHAT_URL = os.getenv("LLAMA_STACK_CHAT_URL", "http://localhost:8321/v1/openai/v1")
 LLM_API_TOKEN = os.getenv("LLM_API_TOKEN", "")
```

Comment:
what is different between `LLAMA_STACK_URL` and `LLAMA_STACK_CHAT_URL`

Reply:
I think the new var got created during the initial debugging. Let me verify by removing the new one and see if it breaks anything (or not).

Reply:
Removed the new var. Using existing `LLAMA_STACK_URL` var.